test/integration/invalid-href/test/index.test.js

```diff
@@ -74,7 +75,8 @@ const noError = async (pathname, click = false) => {
     })
     window.next.router.replace('${pathname}')
   })()`)
-  await waitFor(250)
+  // wait for page to be built and navigated to
+  await waitFor(3000)
```

Comment:
We should probably look into adding a `didHydration` callback or something for testing instead of trying to waiting for an arbitrary amount of time 🤔